#!/usr/bin/env python
PACKAGE = "uav"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("scaleX",		double_t, 0, "scaleX",    	  1, -10, 10);
gen.add("scaleY",		double_t, 0, "scaleY",    	  1, -10, 10);
gen.add("scaleZ",		double_t, 0, "scaleZ",    	  1, -10, 10);
gen.add("scaleYaw",		double_t, 0, "scaleYaw",    	  0.5, -10, 10);
gen.add("disRange",		double_t, 0, "disRange",          5,   0, 30);
gen.add("w",		        int_t,    0, "cols",    	  640, 320, 1280);
gen.add("h",		        int_t,    0, "rows",              600, 320, 1280);

exit(gen.generate(PACKAGE, "uav", "CtrlParam"))
